examples/validation/src/main/kotlin/io/fritz2/examples/validation/validation.kt

```diff
@@ -37,6 +38,7 @@ fun main() {
     val store = object : RootStore<String>(""), Validation<String, ValMsg, String> {
         override val validator = EMailValidator
 
+
         val updateWithValidation = Handler<String> { data, newData ->
```

Comment:
Should we put this (as lazy) inside the Validation-interface, so every validated store inherits if and it can be used without any code?